core/templates/dev/head/components/rule_editor_directive.html

```diff
@@ -28,8 +28,10 @@
     </div>
 
     <div ng-if="rule.type == 'FuzzyMatches'">
-      <classifier-panel rule-inputs="rule.inputs" on-training-data-deletion="onDeleteTrainingDataEntry(index)">
```

Comment:
Well this directive did not exist in the first place. I think it was used as placeholder so that it can be implemented in future.